ElektroniskDodsmelding.Samples/README.md

```diff
@@ -0,0 +1,5 @@
+## Samples run as Tests
+This samples are configured to run Tests and uses the Elektronisk dødsmleding API. Can be run localy, but need a valid jwtToken from a HelseId Client to run successfully.
```

Comment:
**Typos/spelling:**
this -> these
Elektronisk dødsmleding API -> Elektronisk dødsmelding API
localy -> locally 
jwtToken -> JWT / JSON Web Token